.github/workflows/ci.yml

```diff
@@ -40,17 +41,47 @@ jobs:
         run: |
           apk add \
             unzip \
+      - name: Cache mongodb PHP extension build
+        if: (!startsWith(matrix.php, '7.1'))
+        uses: actions/cache@v1
+        with:
+          path: /var/tmp/build/ext-mongodb-${{ env.EXT_MONGODB_VERSION }}
+          key: ext-mongodb-${{ env.EXT_MONGODB_VERSION }}-php${{ matrix.php }}-${{ github.sha }}
```

Comment:
You're right, but this will be hard to achieve here... It'll require another step before this. :x

I think we can live with this for now. Keep things simple.